main.go

```diff
@@ -204,6 +205,48 @@ func main() {
 			Name: "disable-content-adding",
 		},
 	}
+	app.Commands = []*cli.Command{
+		{
+			Name: "setup",
+			Action: func(cctx *cli.Context) error {
+				db, err := setupDatabase(cctx)
+				if err != nil {
+					return err
+				}
+
+				username := "admin"
+				passHash := ""
+
+				if err := db.First(&User{}, "username = ?", username).Error; err == nil {
+					return fmt.Errorf("an admin user already exists")
+				}
+
+				newUser := &User{
+					UUID:     uuid.New().String(),
+					Username: username,
+					PassHash: passHash,
+					Perm:     100,
+				}
+				if err := db.Create(newUser).Error; err != nil {
+					return fmt.Errorf("admin user creation failed: %w", err)
+				}
+
+				authToken := &AuthToken{
+					Token:  "EST" + uuid.New().String() + "ARY",
+					User:   0,
+					Expiry: time.Now().Add(time.Hour * 24 * 365),
+				}
+				if err := db.Create(authToken).Error; err != nil {
+					return fmt.Errorf("admin token creation failed: %w", err)
+				}
+
+				fmt.Printf("Auth Token: %v", authToken.Token)
+
+				return nil
+			},
+			Usage: "Creates an initial auth token under new user \"admin\"",
```

Comment:
nit: put the usage above the action function definition

Reply:
LOL whoops, that's not a nit at all, i committed a crime